src/component/AboutUs.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+  Container,
+  TextContainer,
+  Text,
+  Bold
+} from "../Styles/StyledComponents";
+
+const primary = "#fafafa";
+const secondary = "#00bcd4";
+
+const styles = {
+  containerStyle: {
+    textAlign: "center",
+    display: "flex",
+    flexDirection: "column"
+  },
+  backgroundStyle: {
```

Comment:
it is weird to me that we are using styled components and inline style. I feel like we should not have so many sources of style information.

Reply:
What do you think about using a css file and importing that here instead of having these styles using classNames?